Library/BillingInfo.cs

```diff
@@ -305,6 +309,11 @@ internal override void WriteXml(XmlTextWriter xmlWriter)
 
                 xmlWriter.WriteStringIfValid("vat_number", VatNumber);
 
+                if (!Currency.IsNullOrEmpty())
+                {
+                    xmlWriter.WriteElementString("currency", Currency);
+                }
+

```

Comment:
Could we utilize `WriteStringIfValid` instead? `xmlWriter.WriteStringIfValid("currency", Currency);`